src/main/java/org/broadinstitute/consent/http/db/mapper/FileStorageObjectMapperWithFSOPrefix.java

```diff
@@ -0,0 +1,11 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import org.broadinstitute.consent.http.models.FileStorageObject;
+import org.jdbi.v3.core.mapper.RowMapper;
+
+public class FileStorageObjectMapperWithFSOPrefix extends FileStorageObjectMapper implements RowMapper<FileStorageObject>, RowMapperHelper {
+    @Override
+    public String getPrefix() {
+        return "fso_";
+    }
```

Comment:
the story/logical thread behind this class is that:
- I need to be able to map to a `FileStorageObject` class in the Dataset reducer, so I can get the file objects in the dataset.
- I needed to add the `fso_` prefix, because Dataset also has a column named `create_date`
- Since there is custom logic in the FSO mapper, e.g. getting the `BlobId` from the gcs uri, and finding the category enumeration, I couldn't just use `@RegisterBeanMapper(FileStorageObject.class, prefix="fso")`, since it is a "dumb" mapper
- It seems no other model has had this specific issue, and it's impossible to pass parameters to a mapper. To solve these issues, I made a subclass mapper which uses the `fso_` prefix and the original mapper's logic. In the normal mapper, there is no prefix.

Reply:
Took me a while to get this, thank you - I think this is the best approach given the usage. Conceptually, I would have preferred a way to invoke the mapper with different contexts (i.e. prefixes) in different queries, but I don't think that's technically possible in jdbi.